themes/digital.gov/layouts/shortcodes/do-dont-table.html

```diff
@@ -0,0 +1,43 @@
+{{/* 
+
+{{< do-dont-table caption="This is a table" >}}
+  {{< do-dont-row do="do this today" dont="don't do this today" >}}
+  {{< do-dont-row do="open the door for those who need a hand" dont="use inappropriate language in the chats" >}}
+  {{< do-dont-row do="read the directions before using" dont="forget to wash your hands for 20 seconds" >}}
+{{< /do-dont-table >}}
+
+When setting up a **do-dont-table**, you need to make sure you have the correct formatting.
+
+* Start with a a **do-dont-table** then nest **do-dont-row**
+* When you don't provide a matching pair for **do** or **don't**, the table row will not appear. Make sure you have both.
+* Make sure to close the **do-dont't table** with a closing tag, it is not needed for the **do-dont-row**
+
+*/}}
+
+
+{{- if (.Get "caption") -}}
```

Comment:
Looks like caption is required to render, but it's not mentioned anywhere in instructions.

Reply:
Great catch, notes are updated. I've added the logic to not show it if not provided by accident.